### ISSUE TITLE
Problems running `python3 build.py` in Ubuntu 17.04

### ISSUE THREAD

**Issue by stfnp** [bow-simulation/virtualbow#89]
In GitLab by **ozra** on Jan 8, 2018, 08:33

First I got:
```
Traceback (most recent call last):
  File "build.py", line 66, in <module>
    build_vtk("build/vtk/source", "build/vtk/build", "build/vtk")
TypeError: build_vtk() takes 2 positional arguments but 3 were given
```
Tried to run it again immediately after (_just because_...) and then I got another error:
```
Traceback (most recent call last):
  File "build.py", line 69, in <module>
    build_application(".", "build/bow-simulator/build", "build/bow-simulator")
  File "/home/oscar/3-p/bow-simulator/platforms/linux/build.py", line 28, in build_application
    "-DCMAKE_BUILD_TYPE=Release"])
  File "/usr/lib/python3.5/subprocess.py", line 247, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib/python3.5/subprocess.py", line 676, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.5/subprocess.py", line 1282, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: 'cmake'
```

[edit] I tried a third time and then I got yet another:
```
Traceback (most recent call last):
  File "build.py", line 72, in <module>
    build_packages("0.4", "build/packages/build", "build/packages")
  File "/home/oscar/3-p/bow-simulator/platforms/linux/build.py", line 117, in build_packages
    build_deb_package(version, build_dir + "/build-deb", output_dir)
  File "/home/oscar/3-p/bow-simulator/platforms/linux/build.py", line 60, in build_deb_package
    create_install_tree(build_dir)
  File "/home/oscar/3-p/bow-simulator/platforms/linux/build.py", line 48, in create_install_tree
    shutil.copy("build/bow-simulator/bin/bow-simulator", output_dir + "/usr/local/bin")    # Todo: Repetition
  File "/usr/lib/python3.5/shutil.py", line 241, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib/python3.5/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: 'build/bow-simulator/bin/bow-simulator'
```

And a fourth time where it says nothing, so looks like it's succeeding (it's not)

**Comment by stfnp** [comment]
In GitLab by **ozra** on Jan 8, 2018, 08:38

changed the description

**Comment by stfnp** [comment]
In GitLab by **ozra** on Jan 9, 2018, 12:24

I've started a bit on this during my work-breaks, just so you know (Linux [Ubuntu] side only).

**Comment by stfnp** [comment]
In GitLab by **ozra** on Jan 9, 2018, 13:03

Successfully compiled it now, so I'll PR the changes later.

**Comment by stfnp** [comment]
In GitLab by **spfeifer** on Jan 25, 2018, 23:01

closed via commit 31b275d7e66f5d2fd06dd7615dc4af62b1a724ae

**Comment by stfnp** [comment]
In GitLab by **spfeifer** on Mar 18, 2018, 17:19

mentioned in commit 31b275d7e66f5d2fd06dd7615dc4af62b1a724ae
